Makefile

```diff
@@ -27,6 +27,14 @@ endif
 GO_BUILD := CGO_ENABLED=0 go build -tags=$(BUILDTAGS) -ldflags=$(LDFLAGS)
 GO_TEST := CGO_ENABLED=0 go test -tags=$(BUILDTAGS) -ldflags=$(LDFLAGS)
 
+DEFAULT_NETWORK_INTERFACE := $(shell netstat -r | grep default | awk '{print $$NF}')
```

Comment:
Not sure if the Windows build is currently working but this will definitely break it.

If it is currently working, would be good to disable this on Windows.

Reply:
Implemented a little conditional to by pass it when not on linux